boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py

```diff
@@ -41,25 +41,159 @@
 if not os.path.exists(outdir):
     os.mkdir(outdir)
 
-version_dir = os.path.join(outdir, full_version)
-if not os.path.exists(version_dir):
-    os.mkdir(version_dir)
-
-with open('config/templates/csv-template.yaml'.format(operator_name), 'r') as stream:
-    csv = yaml.load(stream)
-
-csv['spec']['customresourcedefinitions']['owned'] = []
-
-# Copy all CRD files over to the bundle output dir:
-crd_files = [ f for f in os.listdir('deploy/crds') if f.endswith('_crd.yaml') ]
-for file_name in crd_files:
-    full_path = os.path.join('deploy/crds', file_name)
-    if (os.path.isfile(os.path.join('deploy/crds', file_name))):
-        shutil.copy(full_path, os.path.join(version_dir, file_name))
-    # Load CRD so we can use attributes from it
-    with open("deploy/crds/{}".format(file_name), "r") as stream:
-        crd = yaml.load(stream)
-    # Update CSV template customresourcedefinitions key
+VERSION_DIR = os.path.join(outdir, full_version)
+if not os.path.exists(VERSION_DIR):
+    os.mkdir(VERSION_DIR)
+
+with open('config/templates/csv-template.yaml'.format(OPERATOR_NAME), 'r') as stream:
```

Comment:
The `format()` call here does nothing.

Reply:
True story. I commented on that elsewhere as well. Fixed.